src/index.ts

```diff
@@ -77,37 +93,49 @@ class LinkSDK {
   }
 
   // Open My Account to authorize application to use MtLink API
-  authorize(options: MyaccountOptions = {}): void {
-    const { newTab = false, backTo, email, authPage } = options;
-    const { PATHS: { OAUTH }} = MY_ACCOUNT;
-    const configs = {
-      sdk_platform: 'js',
+  authorize(options: MyAccountOptions = {}): void {
+    const { newTab = false, email, authPage, backTo, showAuthToggle } = options;
+
+    const params = encodeConfigWithParams(this.params, {
       email,
+      sdk_platform: 'js',
+      sdk_version: packageJSON.version,
       auth_action: authPage,
       back_to: backTo,
-      show_auth_toggle: true
-    };
-    const params = qs.stringify({ ...this.params, configs: encodeConfig(configs) }, { encode: false });
-    window.open(`https://${this.domains.myaccount}/${OAUTH}?${params}`, newTab ? '_blank' : '_self');
+      show_auth_toggle: showAuthToggle
```

Comment:
Documentation mention the default to be `true`

Reply:
if dev didn't provide that value, there is no need to send that to the server, my-account will assumed it is true when it is not defined.

Reply:
That's pretty bad, need to fix asap

Reply:
what is bad about this? that is the whole point of default value

Reply:
Difference between documentation and code